app/mm-redux/actions/groups.ts

```diff
@@ -171,6 +171,19 @@ export function getGroup(id: string): ActionFunc {
     });
 }
 
+export function getGroups(filterAllowReference: false): ActionFunc {
+    return bindClientFunc({
```

Comment:
Why binding? This is dispatching and not returning the action object used in the `loadChannelsForTeam` action, right? Same with the others

Reply:
Fixed. Not using binding anymore.